src/calculateRentalCost.js

```diff
@@ -1,10 +1,27 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const Price = 40;
```

Comment:
Consider renaming the variable 'Price' to follow JavaScript naming conventions, which typically start variable names with a lowercase letter. A better name might be 'dailyRate' or 'dailyRentalPrice'.